spyder/utils/sourcecode.py

```diff
@@ -56,8 +56,11 @@ def get_os_name_from_eol_chars(eol_chars):
 
 
 def get_eol_chars_from_os_name(os_name):
-    """Return EOL characters from OS name"""
-    for eol_chars, name in EOL_CHARS:
+    """Return EOL characters from OS name.

```

Comment:
I think is a little confusing that It could also accept `CRLF`, `LF`, and `CR`,when they aren't os names

Reply:
So how would you like me to proceed? I'm assuming we'd prefer to not rename the function or the args - I'm sure it's used elsewhere and it might also be called with kwargs sometimes.

I could create a new function, but that would be almost a carbon copy of this one.

Reply:
I'm not sure, maybe the best approach will be to rename it and add kwargs: `get_eol_chars(os_name=None, eol=None)` (but that will break compatibility as you said)

We could also consider adding a function `eol_to_os_name(eol):` that will convert `CRLF`, `LF`, and `CR` in windows linux or mac, but It'll be somehow silly.

@ccordoba12 What do you think?

Reply:
I don't think @dougthor42's work should be to fix this because it's already present. We should leave it for another PR.

Reply:
@rlaverde, other than this, are you good with this PR?

Reply:
>I don't think @dougthor42's work should be to fix this because it's already present. We should leave it for another PR.

But  @dougthor42 changed this  function to also receive CRLF, LF, CR

> other than this, are you good with this PR?

Yes, this is my only complain with this PR, If It's isn't an issue this PR is ready to be merged 😄

Reply:
If we don't like the change to `get_eol_chars_from_os_name`, we can simply go back to c9a002da and things should still work just fine. It'll mean that little hack at `spyder/widgets/editor.py` line 1601 would still exist, but I don't think that's a big deal.

Reply:
@dougthor42, ok, please revert to that commit so we can merge this one. This is the last PR before releasing 4.0 beta1!